packages/@glimmer/syntax/test/parser-node-test.ts

```diff
@@ -285,47 +343,36 @@ test('Element modifiers', () => {
   );
 });
 
-test('Element paths', (assert) => {
```

Comment:
I didn't see a replacement test for this scenario -- why was the test removed? (or did I miss an equiv test?)